### PR TITLE
Reinitializing the terraform for destroy functionality for vSphere platform

### PR DESCRIPTION
**Pull request by vavuthu**
Reinitializing the terraform for destroy functionality for vSphere platform

Signed-off-by: vavuthu <vavuthu@redhat.com>

Issue: #886

### CODE REVIEW

**Comment by vavuthu** [comment]
Destroy Functionality : https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-destroy-ocs-cluster/1079/

File: ocs_ci/deployment/terraform.py [hunk 1]
```diff
@@ -16,12 +16,16 @@ class Terraform(object):
     def __init__(self, path):
         self.path = path
 
-    def initialize(self):
+    def initialize(self, upgrade=None):
         """
         Initialize a working directory containing Terraform configuration files
```
Comment (petr-balogh) [inline ocs_ci/deployment/terraform.py]: Please add it to docstring with explanation. Thanks

Comment (vavuthu) [reply ocs_ci/deployment/terraform.py]: Done

File: ocs_ci/deployment/terraform.py [hunk 2]
```diff
@@ -16,12 +16,21 @@ class Terraform(object):
     def __init__(self, path):
         self.path = path
 
-    def initialize(self):
+    def initialize(self, upgrade=False):
         """
         Initialize a working directory containing Terraform configuration files
+
+        Args:
+            upgrade (boolean): True in case installing modules needs upgrade from
```
Comment (petr-balogh) [inline ocs_ci/deployment/terraform.py]: (bool)

Comment (vavuthu) [reply ocs_ci/deployment/terraform.py]: Done

**Pull request merged**